ui/ui.go

```diff
@@ -144,17 +144,40 @@ func newModel(cfg Config) tea.Model {
 		cfg: cfg,
 	}
 
-	return model{
+	m := model{
 		common: &common,
 		state:  stateShowStash,
 		pager:  newPagerModel(&common),
 		stash:  newStashModel(&common),
 	}
+
+	info, _ := os.Stat(cfg.Path)
+	if info.IsDir() {
+		m.state = stateShowStash
+	} else {
+		m.state = stateShowDocument
+		m.pager.currentDocument = markdown{
+			localPath: cfg.Path,
+			Note:      stripAbsolutePath(cfg.Path, ""),
+			Modtime:   info.ModTime(),
+		}
+	}
+
+	return m
 }
 
 func (m model) Init() tea.Cmd {
 	cmds := []tea.Cmd{m.stash.spinner.Tick}
-	cmds = append(cmds, findLocalFiles(*m.common))
+
+	switch m.state {
+	case stateShowStash:
+		cmds = append(cmds, findLocalFiles(*m.common))
+	case stateShowDocument:
+		content, _ := os.ReadFile(m.common.cfg.Path)
```

Comment:
Same here.

Reply:
I'd guard the content and return an error command